### ISSUE TITLE
Inference on single image

### ISSUE THREAD

**Issue by adamadatasci** [WongKinYiu/ScaledYOLOv4#83]
Hi,
great work !

I want to be familiar with your code. Can you provide a simple example with inference for 1 image ?

Specially this part of the code, so img can just be an png or jpg:

        # Disable gradients
        with torch.no_grad():
            # Run model
            t = time_synchronized()
            inf_out, train_out = model(img, augment=augment)  # inference and training outputs
            #inf_out, train_out = model(img.to(memory_format=torch.channels_last), augment=augment)  # inference and training outputs
            t0 += time_synchronized() - t

            # Compute loss
            if training:  # if model has loss hyperparameters
                loss += compute_loss([x.float() for x in train_out], targets, model)[1][:3]  # GIoU, obj, cls

            # Run NMS
            t = time_synchronized()
            output = non_max_suppression(inf_out, conf_thres=conf_thres, iou_thres=iou_thres, merge=merge)
            t1 += time_synchronized() - t

**Comment by adamadatasci** [comment]
I got it ! Thanks:
!python detect.py --source imm.png --weights yolov4-p7.pt --img-size 1536
